### PR TITLE
[ 01 ] Create login page without logic

### PR DESCRIPTION
**Pull request by GabrielAderaldo**

### CODE REVIEW

**Comment by EdsonHTJ** [comment]
Plese commit in english

File: rolemanager/android/app/src/profile/AndroidManifest.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.rolemanager">
+    package="com.example.teste">
```
Comment (EdsonHTJ) [inline rolemanager/android/app/src/profile/AndroidManifest.xml:2]: Is this right?

File: rolemanager/web/index.html [hunk 2]
```diff
@@ -20,26 +23,82 @@
   <!-- iOS meta tags & icons -->
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
-  <meta name="apple-mobile-web-app-title" content="rolemanager">
+  <meta name="apple-mobile-web-app-title" content="teste">
```
Comment (EdsonHTJ) [inline rolemanager/web/index.html:26]: is this right?

File: rolemanager/lib/View/Styles/inpults.dart [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class Inpults {
+  static Widget inpultClassico(
+      //TODO: Ajeitar o inpult
```
Comment (EdsonHTJ) [inline rolemanager/lib/View/Styles/inpults.dart:5]: please comment in english

File: rolemanager/lib/View/Page/login_page.dart [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+              )),
+        ),
+        Positioned(
+          bottom: 0,
+          left: 20,
+          child: Container(
+              child: Column(
+            children: [
+              Button.bntComFotoUrl(
+                  "https://cdn.discordapp.com/attachments/580125063186087966/933885364391780382/Vector_4.png",
```
Comment (EdsonHTJ) [inline rolemanager/lib/View/Page/login_page.dart:51]: Please separate resources from the source code

**Review by EdsonHTJ — CHANGES_REQUESTED** [review]

**Review by joaopedroalb — COMMENTED** [review]
Create issue to change "Teste" to "Rolemanager"

**Review by EdsonHTJ — APPROVED** [review]

**Pull request merged**